### PR TITLE
Food API requests

### PR DESCRIPTION
**Pull request by phaseharry**
- added validation to each route 
- turned all of the initial seed data into lowercase letters

### CODE REVIEW

File: server/routes/foodRouter.js [hunk 1]
```diff
@@ -10,45 +10,124 @@ const Food = require('../db/models/Food')
...
+  } catch (err) {
+    next(err)
+  }
+})
 
 /*
-  @route          GET api/foods/search/:latitude/:longitude
-  @description    fetches all food within a specified distance from you
+  @route          GET api/foods/search?longitude=40.23232&latitude=74.521322&food=strawberries&range=4.2
+  @description    fetches a specific foods within a specified distance from you
```
Comment (oscarchong) [inline server/routes/foodRouter.js]: no 's' on food

File: server/db/connection.js [hunk 2]
```diff
@@ -5,11 +5,12 @@ const mongoose = require('mongoose')
  * @param {import('express').Express} expressApp 
  */
 const connectToDb = expressApp => {
+  mongoose.set('debug', true)
```
Comment (oscarchong) [inline server/db/connection.js]: debug ????

File: server/routes/foodRouter.js (hunk 1 above)
Comment (phaseharry) [reply server/routes/foodRouter.js]: change incoming

Comment (phaseharry) [reply server/routes/foodRouter.js]: changes incoming

**Pull request merged**